test/e2e/components/vl-tooltip.js

```diff
@@ -50,9 +48,16 @@ class VlTooltip extends VlElement {
         }
     }
 
+    async _getTooltipId() {
+        const parentelement = await this.findElement(By.xpath('..'));
+        return parentelement.getAttribute('aria-describedby');
+    }
+
     async _getTooltipElement() {
         try {
-            return await this.driver.findElement(By.css('.vl-tooltip'));
+            const tooltipId  = await this._getTooltipId();
+            const tooltipElement =  await this.driver.findElement(By.css('#' + tooltipId));
+            return new VlElement(this.driver, tooltipElement);
```

Comment:
Zeer triviaal, maar bovenstaande kan (eventueel) vervangen worden door:
```suggestion
            return new VlElement(this.driver, `#${tooltipId}`);
```